tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -122,7 +122,7 @@ steps:
       targetType: 'inline'
       workingDirectory: ${{ parameters.buildDirectory }}
       script: |
-        flub release setPackageTypesField -g ${{ parameters.tagName }} --types ${{ parameters.packageTypesOverride }} --checkFileExists false
+        flub release setPackageTypesField -g ${{ parameters.tagName }} --types ${{ parameters.packageTypesOverride }} --no-checkFileExists 
```

Comment:
Curious, what's the scenario where we don't want the file check validation?

Reply:
We do not want to check the file validation currently as rollups are generated before the 
Build stage. Currently, we update the types and then build